src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -20,7 +20,7 @@ Permissions specify what level of access your service requires of Sentry resourc
 
 ![Form that allows developer to set what permissions they'll need from their user.](permissions.png)
 
-You cannot create an integration or change an existing integration to have permissions greater than your own user settings.
+You cannot create an integration or change an existing integration to have permissions greater than your own user settings. Permissions are scoped to the organization, so adding scopes for issues and events, projects, or teams will enable access to issues and events, projects, or teams across the organization. 
```

Comment:
I wish we could use parens in normal human language. Maybe something like this would make the sentence less complex?
> Permissions are scoped to the organization, so adding scopes for issues and events, projects, or teams will enable access to **the corresponding features** across the organization.

Reply:
I think this will make more sense when looking at the image (which is not shown in the pr description) but is [here](https://docs.sentry.io/product/integrations/integration-platform/#permissions).